event/event_rest.c

```diff
@@ -520,6 +520,11 @@ int pv_event_rest_recv_done_path(struct evhttp_request *req, const char *path)
 	int ret;
 	off_t size;
 
+	if (pv_event_rest_recv_chunk_path(req, path)) {
```

Comment:
is this to make sure we deplete the chunk receiving queue? was this failing in some case without this?

Reply:
no this just ensures that any content that was loaded to buffer for _done (rather than _chunk) gets written to the file. This makes this callback work for chunked and non chunked transfers also.